### PR TITLE
style: Removed Hardcoded strings

### PR DESCRIPTION
**Pull request by RajaVamsi11**
Fixes [FINCN-33](https://issues.apache.org/jira/browse/FINCN-33?jql=project%20%3D%20FINCN%20AND%20component%20%3D%20fineract-cn-mobile)
Related to #27 
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by RajaVamsi11** [comment]
@therajanmaurya Removed the hard-coded strings that are left after #27. I checked the whole project with options available in Android studio for Hardcoded strings in XML files. There are nothing left.
Can you please review this PR.

File: app/src/main/res/layout/layout_nav_header.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
         android:layout_height="wrap_content"
         android:layout_width="match_parent"
         android:paddingTop="@dimen/nav_header_vertical_spacing"
-        android:text="Rajan Maurya"
+        android:text="@string/rajan_maurya"
```
Comment (therajanmaurya) [inline app/src/main/res/layout/layout_nav_header.xml]: Here should not be my name and email, we will show logged in user name and email. Please make string with name `username = Username` and `email = Email` and change it.

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by RajaVamsi11** [comment]
@therajanmaurya Done.

File: app/src/main/res/layout/layout_nav_header.xml (hunk 1 above)
Comment (RajaVamsi11) [reply app/src/main/res/layout/layout_nav_header.xml]: updated

**Comment by miPlodder** [comment]
@therajanmaurya Looks good.

**Review by therajanmaurya — APPROVED** [review]

**Pull request merged**